### ISSUE TITLE
Matching Two Fields Validation not working for Material 

### ISSUE THREAD

**Issue by cpiffat** [formly-js/ngx-formly#1007]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
When using Material, fieldGroup validation does not show error message when group validation returns false. Validation messages do appear for individual fields. 

**Expected behavior**
Would expect similar behavior to bootstrap example shown in https://formly-js.github.io/ngx-formly/examples/validation/matching-two-fields

**Minimal reproduction of the problem with instructions**
https://stackblitz.com/edit/angular-euvzfe?file=app%2Fapp.component.ts

**What is the motivation / use case for changing the behavior?**
Have several small numeric fields set up in a flexbox that need to be set based each other's values. Would like a validation of the group so that the error message shows across the bottom of the entire row rather than scrunched underneath each numeric field. Also, the error messages are more general than any one field being incorrect since the user can adjust the field of their choice to make the numbers valid.

**Please tell us about your environment:**

* **Angular version:** 5.2.0

* **Browser:** [Chrome 66.0.3359.181 | Edge 42.17134.1.0 ]

* **Language:** [TypeScript 2.5.3]

* **Node (for AoT issues):** `node --version` = 9.4.0

**Comment by aitboudad** [comment]
fixed in `4.5.0` by adding `errorPath: 'passwordConfirm'` to specify the target of a validation error
see https://stackblitz.com/edit/angular-hnea9o

**Comment by cpiffat** [comment]
Thanks for fixing so quickly! Does it only work with angular 6+ and angular/material 6+? I am using 5.2 on this project, so ngx-formly version 3 seems to be the only version I can use.

**Comment by aitboudad** [comment]
yes only in `v6.0`, you can use another approach see the following example https://stackblitz.com/edit/angular-qbwj4j?file=src%2Fapp%2Fapp.component.ts

**Comment by cpiffat** [comment]
That won't really work for what I need to do. I have been trying to get a stackblitz that would imitate our project, but the css is not being honored to show it correctly, even with me adding a number of extra flexboxes and hardcoding the widths. I am including the stackblitz, but the image below is what it should look like. I know it appears that using Bootstrap would make more sense than Material, but this is a small section within a larger Material form.
https://stackblitz.com/edit/angular-hnea9o-ddt4tr?file=src%2Fapp%2Fapp.component.css
![hmlerrors](https://user-images.githubusercontent.com/31994677/41780867-92fbe332-75fb-11e8-89b6-4bab8fc984ec.PNG)
 
I really would like the errors to show below the entire line rather than scrunched just below the numeric field. The easiest way to do this seemed to be putting the error around the fieldGroup. Is there another way to handle that? I didn't see a way to list the errors somewhere else on the formly form. 

Thanks!

**Comment by aitboudad** [comment]
ahh I see in that case you have to create a custom wrapper for validation similar to bootstrap, see https://stackblitz.com/edit/angular-yxzbpc

**Comment by cpiffat** [comment]
Thanks! That's perfect!
